src/rosdiscover/cli.py

```diff
@@ -115,32 +116,38 @@ def _observe(args) -> SystemSummary:
     return summary
 
 
-def _periodic_observe(period: int, args: argparse.Namespace) -> SystemSummary:
+def _periodic_observe(interval: int, args: argparse.Namespace) -> SystemSummary:
```

Comment:
should interval be a float?

Reply:
bit of a nitpick, interval could just be grabbed from `args`.

Reply:
We could. I wanted to make it clear through the signature a bit what was required for it to work.